invoicing/ts-classes/Line.ts

```diff
@@ -0,0 +1,12 @@
+import {YLine} from "invoicing";
+import {YTmoney} from "yupana/YClass";
+
+export class Line extends YLine {
+    cGetTotal(): number {
```

Comment:
@pcrispim @AlexYup @bahatron  Does it still make sense to declare the getter/setter in the XML file with specific name ? Could we just use ```get mySlot``` to declare it ?

Reply:
the only thing is there is no total:  https://github.com/loopsoftware/sample-invoicing/blob/master/invoicing/classes/line.xml#L8 
I guess the line should be uncommented?

Reply:
@lportales not sure about that... the accessors in the generated aren't only to proxy the call to the function defined as getter/setter in xml — it's lazily creating new objects (`new YTref()`, `new YTmoney()`, etc), calling secondary methods (eg, the setter will call `onbeforeset()` and `onafterset()`) and emitting events (eg, if in the setter the call to `onbeforeset()` returns `false` it emits `datachangecanceled`)...
and there's also the computed slots...

- if we allow developers to directly write their accessors, do we still want to intercept those calls to apply our logic? ...then we must figure out a mechanism to do that.
- or we can put all the burden on them — the setters must validate all input instead of relying on a validation function being called (which may exist on not), getters for computed slots do the work they need to.

i can map what the `YClass` getter and setter are doing and we can decide what is required or what we can get from typescript — some of it can definitely be passed on to factories (even in javascript) and make it simpler.

Reply:
Ok for now let's keep the existing mechanism for getter/setter : user methods in the implementation and names in the XML.
I think we can re-challenge that later when we'll have a clearer view of our new implementation.